File: OnlineShopAPI/PaymentInfoMicroservice/PaymentInfo.Api/Models/PaymentRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PaymentInfo.Api.Models
{
    public class PaymentRequest
    {
        public long PaymentId { get; set; }

        public string NameOnCard { get; set; }

        public string CardNumber { get; set; }

        public string SecurityCode { get; set; }

        public DateTime ExpDate { get; set; }

        public string BillingName { get; set; }

        public string BillingUnit { get; set; }

        public string BillingCity { get; set; }

        public string BillingState { get; set; }

        public string BillingZipcode { get; set; }

        public long CardType { get; set; }

        public long AccountId { get; set; }
    }
}

File: OnlineShopUI/OnlineShop/src/app/model/requests/payment-request.ts
export class PaymentRequest {
    paymentId: number;
    nameOnCard: string;
    cardNumber: string;
    securityCode: string;
    expDate: Date;
    billingName: string;
    billingUnit: string;
    billingCity: string;
    billingState: string;
    billingZipcode: string;
    cardType: number;
    accountId: number;

    constructor(paymentId: number, nameOnCard: string, cardNumber: string, securityCode: string, expDate: Date, billingName: string, billingUnit: string, 
        billingCity: string, billingState: string, billingZipcode: string, cardType: number, accountId: number){
        this.paymentId = paymentId;
        this.nameOnCard = nameOnCard;
        this.cardNumber = cardNumber;
        this.securityCode = securityCode;
        this.expDate = expDate;
        this.billingName = billingName;
        this.billingUnit = billingUnit;
        this.billingCity = billingCity;
        this.billingState = billingState;
        this.billingZipcode = billingZipcode;
        this.cardType = cardType;
        this.accountId = accountId;
    }
}
